frontend/views/containers/access/PasswordForm.vue

```diff
@@ -26,13 +46,31 @@ label.field
 </template>
 
 <script>
+import { base58btc } from 'multiformats/bases/base58'
```

Comment:
We have our own library for this, '@chelonia/multiformats/bases/base58'.

You can use it like this: 

```
import { base58btc } from '@chelonia/multiformats/bases/base58'
```

Reply:
Right. I see the project already has `@chelonia/multiformats`. replaced it.